pygmt/src/grdsample.py

```diff
@@ -61,9 +56,10 @@ def grdsample(
     {outgrid}
     {spacing}
     {region}
-    translate : bool
-        Translate between grid and pixel registration; if the input is
-        grid-registered, the output will be pixel-registered and vice-versa.
+    translate
+        Translate between grid and pixel registration; if the input is grid-registered,
+        the output will be pixel-registered and vice-versa. This is a destructive grid
+        change.
```

Comment:
Follow upstream docs at https://docs.generic-mapping-tools.org/6.5/grdsample.html#t

```suggestion
        the output will be pixel-registered and vice-versa. This is a *destructive* grid
        change; see :gmt-docs:`reference/options.html#switch-registrations`.
```